closed/custom/copy/Copy-java.base.gmk

```diff
@@ -59,11 +59,22 @@ $(call openj9_copy_files_and_debuginfos, \
 # CPU targets without JIT support.
 NO_JIT_CPUS := riscv64
 
+# Decide which GC lib(s) to copy
+ifneq (off,$(OMR_MIXED_REFERENCES_MODE))
+  POINTER_MODE := mixed
+else ifeq (compressedrefs,$(OPENJ9_LIBS_SUBDIR))
+  POINTER_MODE := compressed
+else
+  POINTER_MODE := full
+endif
```

Comment:
nit: please end a sentence with a period.

I don't think this handles `OMR_MIXED_REFERENCES_MODE=dynamic` properly. In that case I expect only one GC library (`j9gc29`), but the code below will try to copy the 'full' variants as well.

Reply:
Ah, my understanding was a bit off. Right, in dynamic mode there's no point in generating two libraries because there's no OVERRIDE to set, so there will be no difference in the builds. Will be updating this and fixing it in my OMR/OpenJ9 changes too